styles2.css

```diff
@@ -36,13 +36,31 @@ main {
   display: flex;
   justify-content: space-between;
   width: 100%;
+  position: fixed;
+  background-color: #fff;
 }
 
 a {
   text-decoration: none;
   color: var(--text-color1);
 }
 
+nav#x {
+  font-size: 28px;
+  font-weight: 500;
+  display: none;
+}
+
+div#menu {
+  display: none;
+  width: 100%;
+  background-color: #fff;
+  height: 100px;
+  position: fixed;
+  top: 48px;
+  z-index: 3;
+}
+
```

Comment:
- [ ] Kindly make the background of your mobile menu take the full height of the page to match the given design.
- [ ] Please the correct font size, font weight, and the correct margins for your mobile menu options to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/215631506-948418fb-fce9-4a1b-a2a3-0583b4eaaa3d.png) | ![image](https://user-images.githubusercontent.com/68796232/215631462-89e82f8f-44be-4511-a4b2-f59bd222cc83.png) |